dashboard/app/models/user.rb

```diff
@@ -502,6 +504,27 @@ def sanitize_race_data_set_urm
     true
   end
 
+  # Implement validation that refuses to set admin:true attribute unless
+  # there's a Code.org google sso option present.
+  def enforce_google_sso_for_admin
+    return unless migrated?
```

Comment:
This return means we don't enforce this rule for unmigrated users.  Do we instead want to enforce a rule saying unmigrated users may not be admins?

Reply:
Yes, we shouldn't allow unmigrated users to be admins they (the remaining admins in our system) won't be able to add google as an authentication option to their account unless they are migrated.